logger.js

```diff
@@ -0,0 +1,38 @@
+const { createLogger, format, transports } = require('winston');
+const winstonTimestampColorize = require('winston-timestamp-colorize');
+
+const customFormat = format.combine(
+  format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss' }), 
+  winstonTimestampColorize(), 
+  format.colorize({all:true}), 
+  format.printf(({ level, timestamp, message, stack, tool_name, url }) => {
+  
+    let logMessage = `${level} at ${timestamp}`;
+    logMessage += `\n${message}`;
+      if (tool_name) {
+        logMessage += `\n   Tool: ${tool_name}`;
+      }
+      if (url) {
+        logMessage += `\n   URL: ${url}`;
+      }
+      if (stack) {
+        logMessage += `\n   Stack: ${stack}`;
+      }
```

Comment:
We need to setup custom logger for whole repo, not just for only one particular script

Reply:
okay @akshatnema  it will take some time i will do it in the coming days.Is there anything which i need to take care of from your side.